cmd/config_ci.go

```diff
@@ -91,6 +106,7 @@ func NewConfigCICmd(loaderSaver common.FunctionLoaderSaver, writer ci.WorkflowWr
 
 func runConfigCIGitHub(
 	cmd *cobra.Command,
+	// TODO(twoGiants): common.FunctionLoader is enough
```

Comment:
Was this intended to be changed within this PR or does this need some bigger refactor -- intended for a follow up PR?